src/structures/game_round.ts

```diff
@@ -366,46 +367,60 @@ export default class GameRound extends Round {
      */
     async interactionMarkAnswers(correctGuesses: number): Promise<void> {
         if (!this.interactionMessage) return;
-        try {
-            await this.interactionMessage.edit({
-                embeds: this.interactionMessage.embeds,
-                components: this.interactionComponents.map((x) => ({
-                    type: 1,
-                    components: x.components.map((y) => {
-                        const z = y as Eris.InteractionButton;
-                        const noGuesses =
-                            this.interactionIncorrectAnswerUUIDs[
-                                z.custom_id
-                            ] === 0;
-
-                        let label = z.label;
-                        let style: 1 | 3 | 4;
-                        if (this.interactionCorrectAnswerUUID === z.custom_id) {
-                            if (correctGuesses) {
-                                label += ` (${correctGuesses})`;
-                            }
-
-                            style = 3;
-                        } else if (noGuesses) {
-                            style = 1;
-                        } else {
-                            label += ` (${
-                                this.interactionIncorrectAnswerUUIDs[
-                                    z.custom_id
-                                ]
-                            })`;
-                            style = 4;
-                        }
-
+        const components: Eris.ActionRow[] = this.interactionComponents.map(
+            (x) => ({
+                type: 1,
+                components: x.components.map((y: Eris.InteractionButton) => {
+                    if (
+                        Object.values(ClipAction).includes(
+                            y.custom_id as ClipAction,
+                        )
+                    ) {
+                        // TODO heart here
                         return {
```

Comment:
are we just trying to do a no-op here?

Reply:
Yes, but it needs to be returned, otherwise it will exclude this button on the edit, so it can't be filtered out